<launch>
	<node pkg="rast_corridor_planning" name="map_sim_example" type="map_sim_example" output="screen">
		<rosparam 
	        file="$(find rast_corridor_planning)/config/cfg.yaml"
	        command="load" 
		/> 
	</node>
	<node pkg="rast_corridor_planning" name="planning_node" type="planning_node" output="screen">
		<rosparam 
	        file="$(find rast_corridor_planning)/config/cfg.yaml"
	        command="load" 
		/> 
	</node>
	
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find rast_corridor_planning)/rviz/boxes.rviz"/>
<!--	<node pkg="rviz" type="rviz" name="rviz2" args="-d $(find dynamic_occpuancy_map)/rviz/future_status.rviz"/>-->
</launch>
